### PR TITLE
Fix for issue #10

### PR DESCRIPTION
**Pull request by ngrigoriev**
Check the index status report for the new indexes to make sure they get
registered correctly.

Signed-off-by: Nikolai Grigoriev <ngrigoriev@newforma.com>

### CODE REVIEW

File: src/main/java/com/newforma/titan/schema/SchemaManager.java [hunk 1]
```diff
@@ -878,18 +879,23 @@ private void populateNewIndexes(JanusGraph graph, GraphState graphState) throws
...
 						.call();
 
-				LOG.info("Enabling index {}...", indexName);
-				final JanusGraphManagement mgmtIndexEnabler = graph.openManagement();
-				mgmtIndexEnabler.updateIndex(mgmtIndexEnabler.getGraphIndex(indexName), SchemaAction.ENABLE_INDEX).get();
-				mgmtIndexEnabler.commit();
-				LOG.info("Index {} is enabled, existing data may need to be reindexed", indexName);
-
-				graphState.addIndex(indexName, true);
+				if (indexStatusReport.getSucceeded()) {
```
Comment (vicneanschi) [inline src/main/java/com/newforma/titan/schema/SchemaManager.java:887]: align with closing bracket

File: src/main/java/com/newforma/titan/schema/SchemaManager.java [hunk 2]
```diff
@@ -926,18 +932,23 @@ private void populateNewLocalPropIndexes(JanusGraph graph, GraphState graphState
...
 						.timeout(this.reindexTimeoutInSecs, ChronoUnit.SECONDS)
 						.call();
 
-				LOG.info("Enabling local property index {}...", indexName);
-				final JanusGraphManagement mgmtIndexEnabler = graph.openManagement();
-				mgmtIndexEnabler.updateIndex(mgmtIndexEnabler.getRelationIndex(pk, indexName), SchemaAction.ENABLE_INDEX).get();
-				mgmtIndexEnabler.commit();
-				LOG.info("Local property index {} is enabled, existing data may need to be reindexed", indexName);
+				if (indexStatusReport.getSucceeded()) {
+    				LOG.info("Enabling local property index {}...", indexName);
```
Comment (vicneanschi) [inline src/main/java/com/newforma/titan/schema/SchemaManager.java]: indent "if" body

File: src/main/java/com/newforma/titan/schema/SchemaManager.java [hunk 3]
```diff
@@ -975,18 +986,23 @@ private void populateNewLocalEdgeIndexes(JanusGraph graph, GraphState graphState
...
 						.call();
 
-				LOG.info("Enabling local edge index {}...", indexName);
-				final JanusGraphManagement mgmtIndexEnabler = graph.openManagement();
-				mgmtIndexEnabler.updateIndex(mgmtIndexEnabler.getRelationIndex(el, indexName), SchemaAction.ENABLE_INDEX).get();
-				mgmtIndexEnabler.commit();
-				LOG.info("Local edge index {} is enabled, existing data may need to be reindexed", indexName);
-
-				graphState.addIndex(indexName, true);
+				if (indexStatusReport.getSucceeded()) {
```
Comment (vicneanschi) [inline src/main/java/com/newforma/titan/schema/SchemaManager.java]: fix indentation

**Review by vicneanschi — APPROVED** [review]

**Review by pfilion — APPROVED** [review]

**Comment by ngrigoriev** [comment]
Hrr.....it looks like after my last Eclipse update my formatting settings went to /dev/null! :(

**Pull request merged**